keep/parser/parser.py

```diff
@@ -314,7 +313,7 @@ def _inject_env_variables(self, config):
 
     def _parse_providers_from_workflow(
         self, context_manager: ContextManager, workflow: dict
-    ) -> typing.List[BaseProvider]:
+    ) -> None:
```

Comment:
Why remove type here?

Reply:
I removed the type because the method does not return anything.